doc/installation.md

```diff
@@ -126,6 +126,12 @@ Or install capa with build dependencies:
 
 `$ pip install -e /local/path/to/src[build]`
 
+#### Generate rule cache
+
+Generate cache for all rules in the `rules` folder and save the output in the `cache` folder.
+
+`$ python scripts/cache-ruleset.py rules/ cache/`
+
```

Comment:
this should not be necessary, as the caching subsystem should generate the cache on first use. we do have this script to help package a pre-built cache with the standalone .exe distributed via github.

in theory, our users and devs shouldn't be aware the caching at all. but maybe the abstraction is leaky. did you find any bugs?

Reply:
The problem is that I am running the PyInstaller locally to test if the new binja backend works. If I run the PyInstaller command without generating the rule cache first. The generated executable will complain the rule cache is missing when I run it. Note, I added the above code snippet in the section `3. Compile binary using PyInstaller`

Reply:
ahh, i see. then we should fix the caching layer to better handle running in pyinstaller without the cache being present. thanks for pointing this out!

Reply:
good point on the fact that this is only relevant for "Compile binary using PyInstaller", should be fine!